src/code/z_message.c

```diff
@@ -432,12 +432,12 @@ u32 func_80151C9C(PlayState* play) {
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_message/func_80151DA4.s")
 
 void func_80152434(PlayState* play, u16 arg2) {
-    play->msgCtx.unk12046 = 0;
+    play->msgCtx.blockSunsSong = 0;
```

Comment:
```suggestion
    play->msgCtx.blockSunsSong = false;
```
Could you make `blockSunsSong` bool?